platform/android/app/src/main/kotlin/coop/polypoly/polypod/features/FeatureStorage.kt

```diff
@@ -46,27 +47,22 @@ class FeatureStorage {
 
     fun loadFeature(context: Context, fileName: String): Feature {
         val content = ZipFile(File(getFeaturesDir(context), fileName))
-        val manifest = readManifest(context, content)
+        val manifest = readManifest(content)
         return Feature(fileName, content, manifest)
     }
 
     private fun readManifest(
-        context: Context,
         content: ZipFile
-    ): FeatureManifest {
+    ): FeatureManifest? {
         val manifestEntry = content.getEntry("manifest.json")
         if (manifestEntry == null) {
             logger.warn("Missing manifest for '${content.name}'")
-            return FeatureManifest(
-                null, null, null, null, null, null, null, null
-            )
+            return null
```

Comment:
@JJ This is presumably why manifest was made optional above - previously we would return, logically, an empty manifest - now it's just `null`, presumably because we can't create an empty manifest so easily anymore.